docs/content/latest/architecture/docdb-sharding/tablet-splitting.md

```diff
@@ -195,21 +206,94 @@ Tablet UUID                       Range                                 Leader
 ```
 
-{{< note title="Note" >}}
+{{< note title="Important" >}}
 
-There are a few important things to note here. 
-* The original tablet `9991368c4b85456988303cd65a3c6503` is no longer present in the system. It has been replaced with two new tablets. 
-* The tablet leaders are now spread across two nodes, in order to evenly balance the tablets for a table across the nodes in the cluster.
+* The original tablet `9991368c4b85456988303cd65a3c6503` no longer exists and has been replaced with two new tablets.
+* The tablet leaders are now spread across two nodes in order to evenly balance the tablets for the table across the nodes of the cluster.
 
 {{</note >}}
 
-## Automatic splitting
+## Automatic tablet splitting [BETA]
 
 {{< note title="Note" >}}
 
-This feature is currently in [BETA](../../../../faq/general/#what-is-the-definition-of-the-beta-feature-tag).
+Automatic tablet splitting is currently in [BETA](../../../faq/general/#what-is-the-definition-of-the-beta-feature-tag).
 
 {{< /note >}}
 
-Design for automatic splitting can be reviewed on [GitHub](https://github.com/yugabyte/yugabyte-db/blob/master/architecture/design/docdb-automatic-tablet-splitting.md). While the broader feature is [work-in-progress](https://github.com/yugabyte/yugabyte-db/issues/1004), tablets can be automatically split for a few scenarions starting the v2.2 release. Detailed documentation is coming soon.
+Automatic tablet splitting enables resharding of data in a cluster automatically while online, and transparently to users, when a specified size threshold has been reached.
+
+For details on the architecture design, see [Automatic Re-sharding of Data with Tablet Splitting](https://github.com/yugabyte/yugabyte-db/blob/master/architecture/design/docdb-automatic-tablet-splitting.md). While the broader feature is [work-in-progress](https://github.com/yugabyte/yugabyte-db/issues/1004), tablets can be automatically split for a few scenarios starting in the v2.2 release.
+
+### Enable automatic tablet splitting
+
+To enable automatic tablet splitting, use the `yb-master` [`--tablet_split_size_threshold_bytes`](../../../reference/configuration/yb-master/#tablet-split-size-threshold-bytes) flag to specify the size when tablets should split.
+
+The lower the value for the threshold size, the more tablets will exist with the same amount of data.
+
+### Example using a YCSB workload with automatic tablet splitting
+
+In the following example, a three-node cluster is created and uses a YCSB workload to demonstrate the use of automatic tablet splitting in a YSQL database. For details on using YCSB with YugabyteDB, see the [YCSB](../../../benchmark/ycsb-jdbc/) section in the Benchmark guide.
+
+1. Create a three-node cluster.
+
+```sh
+./bin/yb-ctl --rf=3 create --num_shards_per_tserver=1 --ysql_num_shards_per_tserver=1 --master_flags '"tablet_split_size_threshold=30000000"' --tserver_flags '"memstore_size_mb=10"'
```

Comment:
can put a small note here that we use `memstore_size_mb=10` to force flushing files more frequently and see the feature working more quickly